system/update-rootless.sh

```diff
@@ -0,0 +1,18 @@
+cd /mafia
+
+echo Pulling Updates
+git stash
```

Comment:
git stash without git stash pop? Seems like a mistake. Disregard if not though

Reply:
That makes more sense! Before I was copying to the system-config directory that was added to the gitignore.